### PR TITLE
Add nav bar

### PR DESCRIPTION
**Pull request by ronan-f**
image

### CODE REVIEW

**Review by dustxd — APPROVED** [review]
V E R Y  L I T 🚢

File: pages/_app.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import { UserProvider } from "@auth0/nextjs-auth0";
 import { createMuiTheme, ThemeProvider } from '@material-ui/core/styles';
 import appTheme from '../styles/theme';
+import '../styles/globals.css'
```
Comment (dustxd) [inline pages/_app.js:4]: Is this line needed? I don't see any changes in globals.css in this PR 🤔

Comment (ronan-f) [reply pages/_app.js:4]: We were picking up 8px of margin from somewhere, I think it was the browser default which made me think that these globals weren't being respected. Importing them here means they are actually getting applied and we don't have that random margin creeping in

Comment (dustxd) [reply pages/_app.js:4]: Hmmm interesting, good stuff :)

**Pull request merged**